src/views/PageView.vue

```diff
@@ -1,46 +1,70 @@
 <script setup lang="ts">
 
+import { useRoute } from 'vue-router';
 import { useUserStore } from '../stores/user'
-import { useRoute } from 'vue-router'
-import ModulesView from './ModulesView.vue';
-import TipsView from './TipsView.vue';
-import AccountsView from './AccountsView.vue';
-
-// Get the current route
-const location = useRoute();
+import { useDataStore } from '../stores/data'
+import router from '@/router';
+import AddPopup from '@/components/AddPopup.vue';
 
 // Get the stores
 const userStore = useUserStore()
+const dataStore = useDataStore()
+
+// Get the route
+const route = useRoute()
+
+// Save to firebase
+function saveToFirebase() {
+  if (route.name == 'Modules') dataStore.saveModulesToFirebase()
+  if (route.name == 'Tips') dataStore.saveTipsToFirebase()
+}
 
 </script>
 
 <template>
   <main class="page-view">
-    <section>
+    <section class="side-bar">
       <img src="../assets/logo.png" alt="Logo">
       <nav>
         <RouterLink to="/modules">Modules</RouterLink>
         <RouterLink to="/tips">Astuces</RouterLink>
         <RouterLink to="/accounts" v-if="userStore.isAdmin">Comptes</RouterLink>
       </nav>
     </section>
-    <section>
+    <section class="content">
       <header>
-        <h1 class="title-text">UDSP59 FORMATION</h1>
+        <div>
+          <h1 class="title-style">UDSP59 FORMATION</h1>
+          <h2 v-if="route.path.includes('/module/')" class="subtitle-style">
+            {{ (route.name == 'ModulePart' || route.name == 'ModulePartElement')
+              ? (route.name == 'ModulePartElement')
+                ? dataStore.module.title + '.' + ((+route.params.part) + 1) + '.' + ((+route.params.elt) + 1)
+                : dataStore.module.title + '.' + ((+route.params.part) + 1)
+              : dataStore.module.title
```

Comment:
By default, display the dataStore.module.title and in function of the route name, show the rest of the title